docs/examples/index.md

```diff
@@ -6,3 +6,6 @@
 4. [RAG Repair Jupyter Notebook]( https://github.com/AgentToolkit/agent-lifecycle-toolkit/blob/main/examples/rag_repair.ipynb)
 5. [Silent Review Jupyter Notebook](https://github.com/AgentToolkit/agent-lifecycle-toolkit/blob/main/examples/silent_review.ipynb)
 6. [Policy Guard Jupyter Notebook](https://github.com/AgentToolkit/agent-lifecycle-toolkit/blob/main/examples/policy_guard_example.ipynb)
+7. [Test Case Generation Jupyter Notebook](https://github.com/AgentToolkit/agent-lifecycle-toolkit/blob/main/examples/Test-Case-Generation-Example.ipynb)
+8. [Tool Validation Jupyter Notebook](https://github.com/AgentToolkit/agent-lifecycle-toolkit/blob/main/examples/Tool-Validation-Example.ipynb)
+9. [Tool Enrichment Jupyter Notebook](https://github.com/prerna-agarwal-iitd/agent-lifecycle-toolkit/blob/main/examples/Tool-Enrichment-example.ipynb)
```

Comment:
This should point to `https://github.com/AgentToolkit/agent-lifecycle-toolkit/blob/main/examples/Tool-Enrichment-example.ipynb`